workflows/ampliseq.nf

```diff
@@ -100,17 +108,20 @@ if ( params.dada_ref_taxonomy ) {
     taxlevels = params.dada_assign_taxlevels ? "${params.dada_assign_taxlevels}" :
         params.dada_ref_databases[params.dada_ref_taxonomy]["taxlevels"] ?: ""
 } else { taxlevels = params.dada_assign_taxlevels ? "${params.dada_assign_taxlevels}" : "" }
+if ( params.sintax_ref_taxonomy ) {
+    sintax_taxlevels = params.sintax_ref_databases[params.sintax_ref_taxonomy]["taxlevels"] ?: ""
+}
```

Comment:
I assume if `sintax_taxlevels` is specified also when `!params.sintax_ref_taxonomy`, than the test that currently fail might pass. e.g.
```suggestion
} else { sintax_taxlevels = "" }
```
but please test before (I havent)

Reply:
Yes, thanks! Fixed that and another bug I hadn't noticed before.